database/init_data.sql

```diff
@@ -0,0 +1,40 @@
+--counter
+insert into public.counter(counterid, typeamount, typeids) 
+values
+('01', 3, '{ABC}'),
+('02', 2, '{CD}'),
+('03', 2, '{EF}'),
+('04', 1, '{E}'),
+('05', 4, '{ABCD}');
```

Comment:
There is a small issue here, basically the field `typeids` is an array of strings, where each strings should be the ID of a servicetype (e.g. 'A', 'B', etc). But with this insertion method we're inserting just an array with a single string (in the first case, it would be an array with just a single value "ABC"), instead it should be inserted as:
```
('01', 3, '{"A", "B", "C"}'), 
('02', 2, '{"C", "D"}'),
[ .. and so on .. ]
``` 
so the switch would be from `'{ABC}'` to `'{"A", "B", "C"}'` going from an array containing a single string, to an array containing 3 strings.
Same goes for the other rows.